params:
  image_params:
    image_size: 640
  ros_topics:
    raw_image_topic: "/orb_slam3/keyframe_image"
    segmented_image_topic: "/camera/color/image_segment"
    segmented_image_vis: "/camera/color/image_segment_vis"
  model_params:
    model_name: "FastSAM"
    model_path: "include/FastSAM-s.pt"
    point_prompt: [[0, 0]]
    box_prompt: [[0, 0, 0, 0]]
    text_prompt: ""
    point_label: [0]
    iou: 0.9
    conf: 0.4
    contour: False
